test/caPutJsonLogTest.cpp

```diff
@@ -382,6 +384,17 @@ class JsonParser {
             if (!jsonParser->inArray) {
                 jsonParser->waitingKey = true;
             }
+        }//Metadata
+        else if (logger->isMetadataKey(jsonParser->currentKey)) {
+            std::pair<std::map<std::string, std::string>::iterator, bool> ret;
+            ret = jsonParser->metadata.insert(std::pair<std::string, std::string>(jsonParser->currentKey, std::string(reinterpret_cast<const char *>(stringVal),stringLen)));
```

Comment:
super nitpick but space before `stringLen` if you want to be consistent with existing code, and maybe also break up the line